set.mm

```diff
@@ -509765,111 +510464,379 @@ elements in a zero ring (see ~ 0ringirng ).  (Contributed by Thierry
-        AWVFVVOYPWVJVPVPAVWPNHLWVFWYQUFWYQVLWVFVLVWQVXIVYAUYOYRYQAVYDWYPCUTVYIV
-        WPCUYPVQUYQUYRUAUYSWOXJUYTVUAVYHAVVDHLAVVDYSZAHVWTVWPVXTVYHXFZVUBVUCULP
-        VWJVWMWVQVWKVYMUTVWLWVPVWKVYMIVUEVUFVUGZVUHAPWVBWVQWEPVWJWVQWEJVWNAPWVB
-        VWJWVQAEVWIUPAEWVCVWIUJAWUTVWHDWCAOVWGDWBAOWVNVWGUIAWVMVWFVWDAWUNVWEAVV
-        DHLWUNWYRAWUNYSWYSVUNYLVUIXRZYBYBXRZVPVUJUKWYTYTAEVWIVVDWFXUBVUKVULAEOD
-        UJWVLAOVWGDVUMUNZXUAAIWVTVFVWGXUCVFVYJDVVDXUCIVWGVWEVYKVWOXUCVLVUOVQXTV
-        UPVUQALKVURVVBZVVHVVGUTALWIVFZWYCKLVVEVDURZUTZVVEVXLVFZXUDVXHWYDAWVFKXU
-        FAWVGVVOWVFKWVIWVHQYTAHVVELVDVWCYBYRAHVVEVXLVWCVYAYEXUEWYCXBXUDXUGXUHXB
-        LKVUSXDVUTLKVVAVQYN $.
-    $}
+      algextdeglem.n $e |- N = ( x e. U |-> [ x ] ( P ~QG Z ) ) $.
+      algextdeglem.z $e |- Z = ( `' G " { ( 0g ` L ) } ) $.
+      algextdeglem.q $e |- Q = ( P /s ( P ~QG Z ) ) $.
+      algextdeglem.j $e |- J = ( p e. ( Base ` Q ) |-> U. ( G " p ) ) $.
+      ${
+        $d A p q $.  $d A q r $.  $d B p $.  $d E p q $.  $d E r $.
+        $d F p x $.  $d F q $.  $d G p q $.  $d G x $.  $d J p q $.  $d J x $.
+        $d K p q $.  $d L p q $.  $d L r $.  $d L x $.  $d N x $.  $d O p q $.
+        $d O r $.  $d P p q $.  $d P r $.  $d P x $.  $d Q p q $.  $d Q x $.
+        $d U p $.  $d U x $.  $d Z p $.  $d Z x $.  $d p ph q $.  $d ph r $.
+        $d ph x $.
+        $( Lemma for ~ algextdeg .  (Contributed by Thierry Arnoux,
+           2-Apr-2025.) $)
+        algextdeglem2 $p |- ( ph -> G e. ( P LMHom ( ( subringAlg ` L ) ` F ) )
+          ) $=
+          ( csra cfv csn cun cfldgen cress clmhm clmod wcel clss crn wss csubrg
+          cdr csdrg w3a issdrg sylib simp2d eqid sralmod syl cbs flddrngd csubg
+          co subrgsubg subgss 3syl cirng c0g fldcrngd irngssv sseldd fldgensdrg
+          snssd fldgenssid unssad wa subsubrg biimpar syl12anc lsssra wral cpl1
+          unssd fveq2i eqtri cfield adantr evls1fldgencl ralrimiva evls1maplmhm
+          cv simpr rnmptss reslmhm2b biimpa syl31anc fldgenssv resssra eleqtrrd
+          oveq2d ) AJEIHUNUOUOZHICUPZUQZURVSZUSVSZUTVSZEIMUNUOUOZUTVSAXQVAVBZXT
+          XQVCUOZVBZJVDXTVEZJEXQUTVSVBZJYBVBZAIHVFUOZVBZYDAHVGVBZYKHIUSVSZVGVBZ
+          AIHVHUOZVBZYLYKYNVIUDHIVJVKVLZXQIHXQVMZVNVOAHVPUOZXTIHHXTUSVSZXQYRYSV
+          MZYTVMZAYLXTYJVBZYTVGVBZAXTYOVBYLUUCUUDVIAYSXSHUUAAHUCVQZAIXRYSAYKIHV
+          RUOVBIYSVEYQIHVTYSIHUUAWAWBACYSAHIWCVSYSCAYSHILPHWDUOZUFSUUAUUFVMAHUC
+          WEZYQWFUEWGZWIWSZWHHXTVJVKVLZAUUCYKIXTVEZIYTVFUOVBZUUJYQAIXRXTAYSXSHU
+          UAUUEUUIWJWKZUUCUULYKUUKWLXTIHYTUUBWMWNWOWPACRXGZPUOUOZXTVBZRGWQYGAUU
+          PRGAUUNGVBZWLCYSEGHIUUNPUUAUFELWRUOYMWRUOUGLYMWRSWTXAZUHAHXBVBUUQUCXC
+          AYPUUQUDXCACYSVBUUQUUHXCAUUQXHXDXERGUUOXTJUIXIVOAXQYSEHIGJPCRUFUURUUA
+          UHUUGYQUUHUIYRXFYDYFYGVIYHYIEXQYAJYEXTYAVMYEVMXJXKXLAYCYAEUTAYSXTIHMX
+          BUUATAYSXSHUUAUUEUUIXMUUMUCXNXPXO $.
+
+        $( Lemma for algextdeg .  (Contributed by Thierry Arnoux,
```

Comment:
```suggestion
        $( Lemma for ~ algextdeg .  (Contributed by Thierry Arnoux,
```
same for lem6,7,8

Reply:
Fixed in https://github.com/metamath/set.mm/pull/4746/commits/c42e7d79eba289c072cab906104237b2826cbc6a